OptimoveSDK/Sources/Classes/PreferenceCenter/OptimovePreferenceCenter.swift

```diff
@@ -86,14 +86,29 @@ public class OptimovePreferenceCenter {
         Task {
             do {
                 let request = try createGetPreferencesRequest(for: customerId, with: config)
-                guard let data = try await networkClient?.performAsync(request) else {
-                    throw NetworkError.noData
-                }
-                let preferences = try JSONDecoder().decode(OptimovePC.Preferences.self, from: data)
 
-                DispatchQueue.main.async {
-                    completion(.success, preferences)
+                networkClient?.perform(request) { [self] result in
```

Comment:
if we remove client now, we dont need available(13) all over the place? only in PC

Reply:
Resolved